app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -1,9 +1,21 @@
 class Api::V1::PodcastsController < ApplicationController
+
+    protect_from_forgery unless: -> { request.format.json? }
+
     def index
         render json: Podcast.all
     end
 
+    def create
+        podcast = Podcast.new(name: params["podcast"]["name"], url: params["podcast"]["url"])
```

Comment:
Lets use "strong params" by making a private method and using `params.require(:podcast).permit(:name, :url)` or something to that affect